donate/payments/tasks.py

```diff
@@ -65,7 +154,11 @@ def send_transaction_to_basket(data):
             'donation_url': data['landing_url'],
             'locale': data['locale'],
             'conversion_amount': data.get('settlement_amount', None),
-        }
+    }
+    if settings.DONATION_RECEIPT_METHOD == 'DONATE':
```

Comment:
Checking whether or not we set the donation receipts to be sent from the donate stack. If so, process/send donation receipt. If not, just send to SQS.